plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineOptionsSelectionDialog.java

```diff
@@ -140,7 +140,7 @@ public String getText(Object element) {
         return null;
       }
       String simpleName = Signature.getSimpleName(element.toString());
-      return String.format("%s - %s", simpleName, element.toString());
+      return String.format("%s - %s", simpleName, element.toString()); //$NON-NLS-1$
```

Comment:
You can enable RTL with _Preferences > General > Globalization_ (easier to do from a debug instance).  In this case, I'm honestly not sure as the result will be filling in type names which won't be RTL'd.  I just checked the JDT _Open Type_ dialog and it doesn't do it.  So I'll retract this.